### ISSUE TITLE
`gulp dist` raises an error

### ISSUE THREAD

**Issue by mrrooijen** [tinkertrain/jspm-react#10]
```
jspm-react[master] $ gulp dist
[15:34:48] Using gulpfile ~/Desktop/jspm-react/gulpfile.js
[15:34:49] Starting 'dist'...
[15:34:49] Starting 'delete-dist'...
[15:34:49] Finished 'delete-dist' after 8.25 ms
[15:34:49] Starting 'js'...
[15:34:49] Starting 'sass'...
[15:34:49] Finished 'dist' after 599 ms
[15:34:49] null
[15:34:49] Finished 'sass' after 630 ms
[15:34:49] Finished 'js' after 729 ms
[15:34:49] Starting 'build'...
[15:34:49] Finished 'build' after 4.98 μs
[15:34:49] Starting 'css'...
[15:34:49] Starting 'html'...
[15:34:49] Finished 'html' after 1.9 ms
[15:34:49] Starting 'js'...
[15:34:49] Finished 'js' after 28 ms
[15:34:49] Starting 'bundle'...
[15:34:49] Finished 'css' after 200 ms
     Building the single-file sfx bundle for build/js/main.js!jsx...

err  TypeError: Cannot read property 'suppressDeprecationMessages' of undefined
         at Logger.deprecate (file:/Users/Michael/Desktop/jspm-react/jspm_packages/npm/babel-core@5.5.4/browser.js:8889:24)
         at File.normalizeOptions (file:/Users/Michael/Desktop/jspm-react/jspm_packages/npm/babel-core@5.5.4/browser.js:8271:18)
         at new File (file:/Users/Michael/Desktop/jspm-react/jspm_packages/npm/babel-core@5.5.4/browser.js:8237:22)
         at TransformerPipeline.transform (file:/Users/Michael/Desktop/jspm-react/jspm_packages/npm/babel-core@5.5.4/browser.js:12107:16)
         at /usr/local/lib/node_modules/jspm/node_modules/systemjs-builder/compilers/es6.js:125:31
         at tryCatchReject (/usr/local/lib/node_modules/jspm/node_modules/systemjs/node_modules/es6-module-loader/dist/es6-module-loader.src.js:1183:30)
         at runContinuation1 (/usr/local/lib/node_modules/jspm/node_modules/systemjs/node_modules/es6-module-loader/dist/es6-module-loader.src.js:1142:4)
         at Fulfilled.when (/usr/local/lib/node_modules/jspm/node_modules/systemjs/node_modules/es6-module-loader/dist/es6-module-loader.src.js:930:4)
         at Pending.run (/usr/local/lib/node_modules/jspm/node_modules/systemjs/node_modules/es6-module-loader/dist/es6-module-loader.src.js:821:13)
         at Scheduler._drain (/usr/local/lib/node_modules/jspm/node_modules/systemjs/node_modules/es6-module-loader/dist/es6-module-loader.src.js:97:19)
         at Scheduler.drain (/usr/local/lib/node_modules/jspm/node_modules/systemjs/node_modules/es6-module-loader/dist/es6-module-loader.src.js:62:9)
[15:34:55] 'bundle' errored after 5.89 s
[15:34:55] Error in plugin 'gulp-shell'
Message:
    Command failed: /bin/sh -c jspm bundle-sfx build/js/main.js!jsx dist/js/app.js

Details:
    killed: false
    code: 1
    signal: null
    cmd: /bin/sh -c jspm bundle-sfx build/js/main.js!jsx dist/js/app.js
    stdout:      Building the single-file sfx bundle for build/js/main.js!jsx...

err  TypeError: Cannot read property 'suppressDeprecationMessages' of undefined
         at Logger.deprecate (file:/Users/Michael/Desktop/jspm-react/jspm_packages/npm/babel-core@5.5.4/browser.js:8889:24)
         at File.normalizeOptions (file:/Users/Michael/Desktop/jspm-react/jspm_packages/npm/babel-core@5.5.4/browser.js:8271:18)
         at new File (file:/Users/Michael/Desktop/jspm-react/jspm_packages/npm/babel-core@5.5.4/browser.js:8237:22)
         at TransformerPipeline.transform (file:/Users/Michael/Desktop/jspm-react/jspm_packages/npm/babel-core@5.5.4/browser.js:12107:16)
         at /usr/local/lib/node_modules/jspm/node_modules/systemjs-builder/compilers/es6.js:125:31
         at tryCatchReject (/usr/local/lib/node_modules/jspm/node_modules/systemjs/node_modules/es6-module-loader/dist/es6-module-loader.src.js:1183:30)
         at runContinuation1 (/usr/local/lib/node_modules/jspm/node_modules/systemjs/node_modules/es6-module-loader/dist/es6-module-loader.src.js:1142:4)
         at Fulfilled.when (/usr/local/lib/node_modules/jspm/node_modules/systemjs/node_modules/es6-module-loader/dist/es6-module-loader.src.js:930:4)
         at Pending.run (/usr/local/lib/node_modules/jspm/node_modules/systemjs/node_modules/es6-module-loader/dist/es6-module-loader.src.js:821:13)
         at Scheduler._drain (/usr/local/lib/node_modules/jspm/node_modules/systemjs/node_modules/es6-module-loader/dist/es6-module-loader.src.js:97:19)
         at Scheduler.drain (/usr/local/lib/node_modules/jspm/node_modules/systemjs/node_modules/es6-module-loader/dist/es6-module-loader.src.js:62:9)

    stderr:
Error running task sequence: { task: 'bundle',
  message: 'bundle stream',
  duration: 5.894989118,
  hrDuration: [ 5, 894989118 ],
  err:
   { [Error: Command failed: /bin/sh -c jspm bundle-sfx build/js/main.js!jsx dist/js/app.js
   ]
     killed: false,
     code: 1,
     signal: null,
     cmd: '/bin/sh -c jspm bundle-sfx build/js/main.js!jsx dist/js/app.js',
     stdout: '     Building the single-file sfx bundle for build/js/main.js!jsx...\n\nerr  TypeError: Cannot read property \'suppressDeprecationMessages\' of undefined\n         at Logger.deprecate (file:/Users/Michael/Desktop/jspm-react/jspm_packages/npm/babel-core@5.5.4/browser.js:8889:24)\n         at File.normalizeOptions (file:/Users/Michael/Desktop/jspm-react/jspm_packages/npm/babel-core@5.5.4/browser.js:8271:18)\n         at new File (file:/Users/Michael/Desktop/jspm-react/jspm_packages/npm/babel-core@5.5.4/browser.js:8237:22)\n         at TransformerPipeline.transform (file:/Users/Michael/Desktop/jspm-react/jspm_packages/npm/babel-core@5.5.4/browser.js:12107:16)\n         at /usr/local/lib/node_modules/jspm/node_modules/systemjs-builder/compilers/es6.js:125:31\n         at tryCatchReject (/usr/local/lib/node_modules/jspm/node_modules/systemjs/node_modules/es6-module-loader/dist/es6-module-loader.src.js:1183:30)\n         at runContinuation1 (/usr/local/lib/node_modules/jspm/node_modules/systemjs/node_modules/es6-module-loader/dist/es6-module-loader.src.js:1142:4)\n         at Fulfilled.when (/usr/local/lib/node_modules/jspm/node_modules/systemjs/node_modules/es6-module-loader/dist/es6-module-loader.src.js:930:4)\n         at Pending.run (/usr/local/lib/node_modules/jspm/node_modules/systemjs/node_modules/es6-module-loader/dist/es6-module-loader.src.js:821:13)\n         at Scheduler._drain (/usr/local/lib/node_modules/jspm/node_modules/systemjs/node_modules/es6-module-loader/dist/es6-module-loader.src.js:97:19)\n         at Scheduler.drain (/usr/local/lib/node_modules/jspm/node_modules/systemjs/node_modules/es6-module-loader/dist/es6-module-loader.src.js:62:9)\n',
     stderr: '',
     name: 'Error',
     message: 'Command failed: /bin/sh -c jspm bundle-sfx build/js/main.js!jsx dist/js/app.js\n',
     stack: 'Error: Command failed: /bin/sh -c jspm bundle-sfx build/js/main.js!jsx dist/js/app.js\n\n    at ChildProcess.exithandler (child_process.js:751:12)\n    at ChildProcess.emit (events.js:110:17)\n    at maybeClose (child_process.js:1015:16)\n    at Socket.<anonymous> (child_process.js:1183:11)\n    at Socket.emit (events.js:107:17)\n    at Pipe.close (net.js:485:12)',
     showStack: false,
     showProperties: true,
     plugin: 'gulp-shell' } }
```

What I did:
- Clone the repo
- Install jspm `npm install -g jspm`
- Install gulp `npm install -g gulp`
- `npm install`
- `jspm install`
- `gulp dist`

**Comment by nickytonline** [comment]
@meskyanichi, it got fixed by babel in v. 5.7 see the fix babel/babel#1992

**Comment by mrrooijen** [comment]
@nickytonline :+1:
